### PR TITLE
UserDatabase Migration Tests: Part 8

### PR DESCRIPTION
**Pull request by mejdoo**
Part 8 of #2699

Implemented integration tests for **Clients**, **PhoneNumbers** and **Email Addresses** tables for migration from version 126 to 127.

https://wearezeta.atlassian.net/browse/AN-6745
#### APK
[Download build #1630](http://10.10.124.11:8080/job/Pull%20Request%20Builder/1630/artifact/build/artifact/wire-dev-PR2717-1630.apk)
[Download build #1633](http://10.10.124.11:8080/job/Pull%20Request%20Builder/1633/artifact/build/artifact/wire-dev-PR2717-1633.apk)

### CODE REVIEW

File: storage/src/androidTest/kotlin/com/waz/zclient/storage/userdatabase/email/EmailAddressesTable126to127MigrationTest.kt [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+            contactId = contactId,
+            email = email,
+            openHelper = testOpenHelper
+        )
+
+        validateMigration()
+
+        runBlocking {
+            val syncJob = allEmailAddresses()[0]
+            with(syncJob) {
```
Comment (BradleyWire) [inline storage/src/androidTest/kotlin/com/waz/zclient/storage/userdatabase/email/EmailAddressesTable126to127MigrationTest.kt]: same here and every other test

File: storage/src/androidTest/kotlin/com/waz/zclient/storage/userdatabase/email/EmailAddressesTable126to127MigrationTest.kt [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+        getUserDatabase(
+            getApplicationContext(),
+            TEST_DB_NAME,
+            UserDatabase.migrations
+        )
+
+    private suspend fun allEmailAddresses() =
+        getUserDb().emailAddressesDao().allEmailAddresses()
+
+    companion object {
```
Comment (BradleyWire) [inline storage/src/androidTest/kotlin/com/waz/zclient/storage/userdatabase/email/EmailAddressesTable126to127MigrationTest.kt]: I'm thinking we could probably encapsulate the `TEST_DB_NAME` in the parent `UserDatabaseMigrationTest`. The version may need to be dynamic for the migration test helpers (for future room migrations)

Comment (mejdoo) [reply storage/src/androidTest/kotlin/com/waz/zclient/storage/userdatabase/email/EmailAddressesTable126to127MigrationTest.kt]: let me finish all the tables, I do some refactoring a in separate PR

**Comment by mejdoo** [comment]
build again

**Pull request merged**